lib/dma.c

```diff
@@ -351,7 +394,9 @@ MOCK_DEFINE(dma_controller_add_region)(dma_controller_t *dma,
                 vfu_log(dma->vfu_ctx, LOG_WARNING,
                         "failed to close fd %d: %m", region->fd);
             }
-
+            if (region->dirty_bitmap != NULL) {
```

Comment:
nit, don't need to check NULL before a free()

Reply:
ok